packages/v4/patternfly-docs/content/design-guidelines/components/form/forms.md

```diff
@@ -210,7 +212,7 @@ Align helper text with the specific input for which it offers assistance. If you
 
 For more information about writing effective helper text, see the [content](#content) section.
 
-#### Popovers
+##### Popovers
 
 Use a [popover](/components/popover/design-guidelines) for fields that might require additional background or explanation. Popovers may also link to external help pages or other related documentation. 
 
```

Comment:
Bold "use popovers to"

Reply:
Actually re-reading this, you have two "Use a popover" sentences, so I would unbold this and maybe change the structure so it says "Popovers are useful for fields that might require...."

and then believe, bold "Use a popover when:"

Reply:
@wolfeallison missed this comment